docker/release/dockercomposefiles/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3'
+version: '4'
```

Comment:
This is compose file version not the version of changes:
https://docs.docker.com/compose/compose-file/compose-versioning/

Please keep it at version 3 for now.

Thanks.